x-pack/plugins/reporting/server/lib/validate_browser.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import * as puppeteer from 'puppeteer-core';
+import { CHROMIUM } from '../browsers/browser_types';
+
+export const validateBrowser = async (browserFactory: any, log: (message: string) => any) => {
+  if (browserFactory.type === CHROMIUM) {
```

Comment:
This check won't be needed for 7.0+. Is it added here so it can be backported?

Reply:
Yeah, just wanted to make sure it was "type-safe" on the browser